src/components/Footer.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useContext } from 'react';
+import cn from 'classnames';
+import { Status } from '../Types/Status';
+import { DispatchContext, StateContext } from './TodosContext';
+import { deleteTodo } from '../api/todos';
+
+export const Footer: React.FC = () => {
+  const dispatch = useContext(DispatchContext);
+  const { todos, filterBy } = useContext(StateContext);
+
+  const activeTodos = todos.filter(todo => !todo.completed);
+  const completedTodos = todos.filter(todo => todo.completed);
+
+  const handleFilter = (status: Status) => {
+    dispatch({
+      type: 'filterBy',
+      payload: status,
+    });
+  };
+
+  const handleClearCompleted = () => {
+    completedTodos.forEach(todo => {
+      deleteTodo(todo.id)
+        .then(() => {
+          dispatch({
+            type: 'deleteTodo',
+            payload: todo.id,
+          });
+        })
+        .catch(() => {
+          dispatch({
+            type: 'hasError',
+            payload: true,
+          });
+          dispatch({
+            type: 'errorMessage',
+            payload: 'Unable to delete a todo',
+          });
+        });
+    });
+  };
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeTodos.length > 0 ? activeTodos.length : '0'} items left
```

Comment:
You can write it like this
```suggestion
        {activeTodos.length || 0} items left
```